src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -295,7 +295,7 @@ def _to_vespa_tensor_query(self, marqo_query: MarqoTensorQuery) -> Dict[str, Any
 
         if not marqo_query.approximate:
             query['ranking.softtimeout.enable'] = False
-            query['timeout'] = '300s'
+            query['timeout'] = 300 * 1000  # 5 minutes
```

Comment:
FYI you change the value type from `str` to `int`. If the test can pass, then it's fine.

Reply:
The client is making a string with `ms` suffix, that's why